pygameMenu/baseimage.py

```diff
@@ -30,21 +30,33 @@
 -------------------------------------------------------------------------------
 """
 
-import os.path as _path
+import os.path as path
 import pygame
+import math
+
+from pygameMenu.utils import assert_vector2
 
 # https://www.pygame.org/docs/ref/image.html
 _VALID_IMAGE_FORMATS = ['.jpg', '.png', '.gif', '.bmp', '.pcx', '.tga', '.tif', '.lbm',
                         '.pbm', '.pgm', '.ppm', '.xpm']
 
 # Available images
-__actualpath = str(_path.abspath(_path.dirname(__file__))).replace('\\', '/')
+__actualpath = str(path.abspath(path.dirname(__file__))).replace('\\', '/')
```

Comment:
- `.replace('\\', '/')` should not be necessary, the `os.path` library automatically use the correct path separator depending on the OS.
- The `str` is useless.
- The `abspath` should be applied before `dirname` as the content of the var `__file__` may vary depending on the OS
- don't concatenate path yourself, use the `join` method which acts depending on the OS

Use:
```
import os.path as osp
FONTDIR = osp.join(osp.dirname(osp.abspath(__file__)), '{0}', 'resources', 'images', '{1}')
```

Reply:
nice ⭐